devlib/collector/perf.py

```diff
@@ -126,6 +126,10 @@ def __init__(self,
             self.command = command
         else:
             raise ValueError('Unsupported perf command, must be stat or record')
+        if report_options and (command != 'record'):
+            self.logger.warning('report_options specified, but command is not record.  report_options will be ignored')
```

Comment:
Should probably raise an error here, as this is something that the can correct.

Reply:
I have changed it to raise a `ValueError`